<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDialogClass</class>
 <widget class="QDialog" name="InputDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>偏好设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="mainwindow.qrc">
      <normaloff>:/MainWindow/close_black.png</normaloff>:/MainWindow/close_black.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>起点：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="longitudeLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>经度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="latitudeLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>纬度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="locateButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="mainwindow.qrc">
      <normaloff>:/MainWindow/locate.png</normaloff>:/MainWindow/locate.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>偏好：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>68</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>性价比高</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton1">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>距离短</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton2">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>道路畅通</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>时间短</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{border-width:1px;border-style:solid;border-color:#888888;border-radius:2px 2px 2px 2px}
QComboBox::drop-down{border-width:1px;border-style:solid;border-left-color:#888888;}
QComboBox::down-arrow{image:url(:/MainWindow/down_arrow.png);width:12px;}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCategoryButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="mainwindow.qrc">
      <normaloff>:/MainWindow/add.png</normaloff>:/MainWindow/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:#7fa8f3;border:none;color:white;font-size:14px;border-radius:4px 4px 4px 4px}
QPushButton:hover{background-color:#96b6f2;}
QPushButton:pressed{background-color:#b4ccfa;}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:#7fa8f3;border:none;color:white;font-size:14px;border-radius:4px 4px 4px 4px}
QPushButton:hover{background-color:#96b6f2;}
QPushButton:pressed{background-color:#b4ccfa;}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
